Makefile

```diff
@@ -42,7 +44,8 @@ generate-accounts:
 	@ echo "           # this is also a test comment, :O" >> test/config/postfix-accounts.cf
 
 tests:
-	@ NAME=$(NAME) ./test/bats/bin/bats test/*.bats
+# @ NAME=$(NAME) ./test/bats/bin/bats test/*.bats
+	@ NAME=$(NAME) ./test/bats/bin/bats test/open_dkim.bats
```

Comment:
You did this to shorten up the CI it think?

Reply:
Yes, see the PR description

Reply:
My bad ;)